tests/ccl_test_correlation.c

```diff
@@ -58,9 +58,9 @@ static void compare_corr(char *compare_type,int algorithm,struct corrs_data * da
 
   double epsrel_save;
   if(!strcmp(compare_type,"histo")) { //This is needed for the histogrammed N(z) in order to pass the IA tests
-    epsrel_save=ccl_gsl->INTEGRATION_LIMBER_EPSREL;
-    ccl_gsl->INTEGRATION_LIMBER_EPSREL=2.5E-5;
-    ccl_gsl->INTEGRATION_EPSREL=2.5E-5;
+    epsrel_save = cosmo->gsl_params.INTEGRATION_LIMBER_EPSREL;

```

Comment:
Since these are now reinitialized for each cosmology, I think we don't need to save them...

Reply:
fixed